packages/trace-viewer/src/embedded.tsx

```diff
@@ -45,7 +45,7 @@ import { EmbeddedWorkbenchLoader } from './ui/embeddedWorkbenchLoader';
   if (window.location.protocol !== 'file:') {
     if (!navigator.serviceWorker)
       throw new Error(`Service workers are not supported.\nMake sure to serve the Trace Viewer (${window.location}) via HTTPS or localhost.`);
-    navigator.serviceWorker.register('sw.bundle.js');
+    navigator.serviceWorker.register('sw.bundle.js' + window.location.search);
```

Comment:
This breaks the service worker lifecycle, we can't ship it like this. Instead, we can pass the test server port via the trace URL.